src/configmanager.cpp

```diff
@@ -108,13 +108,15 @@ bool ConfigManager::load()
 		string[MYSQL_PASS] = getGlobalString(L, "mysqlPass", "");
 		string[MYSQL_DB] = getGlobalString(L, "mysqlDatabase", "forgottenserver");
 		string[MYSQL_SOCK] = getGlobalString(L, "mysqlSock", "");
+		string[HTTP_API_IP] = getGlobalString(L, "httpApiIp", "127.0.0.1");
```

Comment:
I agree with @DSpeichert.
I think that if someone is building such "advanced" setups then they should have the sanity not to do a something that's comparable in stupidity to leaving a default root password and root login enabled in ssh.